parser/src/parser/tests/boundary_constraints.rs

```diff
@@ -11,11 +11,11 @@ const BASE_MODULE: &str = r#"
 def test
 
 trace_columns {
-    main: [clk]
+    main: [clk];
```

Comment:
@bobbinth Perhaps it would make more sense to use `,` with these top-level definitions? Semicolons feels a bit bizarre in what amounts to a data structure definition.

In statement blocks and stuff, it feels pretty natural, but less so here.

Reply:
Agreed - in declarations like these, commas would be a more natural choice.